qemistree/test-integration-workflow/test_tasks.csv

```diff
@@ -1,4 +1,4 @@
 task_id,description,regressioncountviews
 0ed967411edd4134a5fc5ba0a762ce19,qemistree small test,summary_view
 054a193702f543279e4e5342ac32bdc5,qemistree on foodmics mini,summary_view
-eead2b3fb5cc48639b849110cf98a095,qemistree on serum data,summary_view
\ No newline at end of file
+eead2b3fb5cc48639b849110cf98a095,qemistree on serum data,summary_view
```

Comment:
Are these updates intentional?

Reply:
I believed they were present in the main repo. Let me remove them.